.agents/skills/advantagekit-replay/scripts/compare_replay_topics.py

```
@@ -0,0 +1,163 @@
+    if not roots:
+        raise SystemExit("No WPILib install found under ~/wpilib")
+    return roots[0]
+
+
+def _version_key(name: str) -> tuple[int, ...]:
+    key: list[int] = []
+    for part in name.split("."):
+        for token in re.split(r"[-_]", part):
+            if token.isdigit():
+                key.append(int(token))
+            elif token == "beta":
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to: "token"

(S105)